tools/projmgr/test/src/ProjMgrUnitTests.cpp

```diff
@@ -3351,3 +3351,22 @@ solution:\n\
   }
 }
 
+/*
+* This test case runs the ProjMgrYamlEmitter with a solution
+* that references a project using different file name case.
+*/
+TEST_F(ProjMgrUnitTests, RunProjMgr_YamlEmitterFileCaseIssue) {
+  char* argv[7];
+  const string& csolution = testinput_folder + "/TestSolution/FilenameCase/filenbame.csolution.yml";
+  argv[1] = (char*)"convert";
+  argv[2] = (char*)csolution.c_str();
+  argv[5] = (char*)"-o";
+  argv[6] = (char*)testoutput_folder.c_str();
+  EXPECT_EQ(0, RunProjMgr(7, argv, 0));
```

Comment:
```suggestion
  EXPECT_EQ(1, RunProjMgr(7, argv, 0));
```

If we want to make sure the projects are portable I believe we should throw an error in this case, we should discuss about it.

Reply:
Yes, we need to discuss if we accept different file name case. Otherwise we need to detect this as an issue in validation phase and stop gracefully.

Reply:
We agreed to not allow mismatch in file name cases since we aim for portability.